Monal/Classes/MLXMPPManager.h

```diff
@@ -104,9 +105,10 @@ NS_ASSUME_NONNULL_BEGIN
 -(void) updatePassword:(NSString*) password forAccount:(NSNumber*) accountID;
 
 /**
-Sends a message to a specified contact in account. Calls completion handler on success or failure.
+ Sends a message to a specified contact in account.
+ It returns an MLMessage object (corresponding to the sent message) on success, and nil on failure.
  */
--(void) sendMessageAndAddToHistory:(NSString*) message havingType:(NSString*) messageType toContact:(MLContact*) contact isEncrypted:(BOOL) encrypted uploadInfo:(NSDictionary* _Nullable) uploadInfo withCompletionHandler:(void (^ _Nullable)(BOOL success, NSString* messageId)) completion;
+-(MLMessage* _Nullable) sendMessageAndAddToHistory:(NSString*) message havingType:(NSString*) messageType toContact:(MLContact*) contact isEncrypted:(BOOL) encrypted uploadInfo:(NSDictionary* _Nullable) uploadInfo NS_SWIFT_NAME(sendMessageAndAddToHistory(message:havingType:toContact:isEncrypted:uploadInfo:));
```

Comment:
Normally the NS_SWIFT_NAME is added on top of the prototype, not to its end.

```suggestion
NS_SWIFT_NAME(sendMessageAndAddToHistory(message:havingType:toContact:isEncrypted:uploadInfo:))
-(MLMessage* _Nullable) sendMessageAndAddToHistory:(NSString*) message havingType:(NSString*) messageType toContact:(MLContact*) contact isEncrypted:(BOOL) encrypted uploadInfo:(NSDictionary* _Nullable) uploadInfo;
```

Reply:
Oh, forget it, that was the KSCrash project where I'm currently trying to get some changes upstream. Our codebase uses the same "convention" as you did (add it to the end). "convention", because there is only one instance where `NS_SWIFT_NAME` is used :D

Reply:
Apple puts NS_SWIFT_NAME before for classes, and after for everything else
https://developer.apple.com/documentation/swift/renaming-objective-c-apis-for-swift